### DISCUSSION TITLE
Move the curve graph with the stick.

### DISCUSSION THREAD

**Discussion by robertobrazill** [FrSkyRC/ETHOS-Feedback-Community#3473] [Ideas]
A very useful feature when configuring a new curve is the movement of the graph when using the stick, which is currently static. On JETTI radios, this graphic has movement, which facilitates fine adjustment. On our radio we do it without much reference, as the graphic doesn't move, preventing us from knowing exactly where the radio stick is.

**Comment by bsongis-frsky** [comment]
There is an open issue on the project for that feature. It should be possible to do that soon.

**Reply by robertobrazill** [reply]
TKS BRO.

**Reply by robertobrazill** [reply]
Don't forget this topic, it's really very important for those who use planes with many servos on the aeleron and elevator.
